packages/integration-objects/srv/utils/RateExtensionConstants.js

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+/* eslint-disable max-len */
+const RateExtensionConstants = {
+  FROM_AND_TO_CURRENCY_ARE_SAME: "The 'From Currency' and 'To Currency' must be different from each other.",
```

Comment:
Please also make an issue to go over the modules containing just these constants and check where and how often they are used. In many cases they can be moved to the smaller methods. Just create an issue. Do not do this on this PR.

Reply:
Updated the issue.